docs/publications.md

```diff
@@ -0,0 +1,98 @@
+[FuzzBench](https://dl.acm.org/doi/pdf/10.1145/3468264.3473932)
+[[BibTeX](https://ieeexplore.ieee.org/abstract/document/9787836)]
```

Comment:
@Alan32Liu This is pointing to a different paper.

Reply:
Thanks! Fixing now : )

Reply:
Fixed in #1569.